.codex/skills/README.md

```
@@ -0,0 +1,18 @@
+# Repo-Local Skills
+
+## Table of Contents
+- [Overview](#overview)
+- [Skills](#skills)
+
+## Overview
+
+These skills are specific to `~/dev/diagram-cli` and cover recurring governance and startup failure patterns that the global shared skill library does not own.
+
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)